web/src/main/java/com/epam/esm/controller/TagController.java

```diff
@@ -57,12 +73,10 @@ public TagController(TagService tagService) {
         return ResponseEntity.noContent().build();
     }
 
-    @PostMapping(consumes = "application/json")
-    public ResponseEntity<TagDto> createTag(@RequestBody @Validated({PatchGroup.class, UpdateGroup.class}) @Valid TagDto tagDto) {
-        TagDto savedTag = tagService.save(tagDto);
-        return ResponseEntity.ok(savedTag);
-
+    @GetMapping("/max")
+    public ResponseEntity<TagDto> getUserWithMaxOrderCost() {
```

Comment:
Rename the method because it is called getUser... but it returns Tag.

Reply:
Commit 617b80b